docs/semgrep-appsec-platform/dashboard-beta.md

```diff
@@ -0,0 +1,217 @@
+---
+slug: dashboard-beta
+title: Dashboard (beta)
+hide_title: true
+description: Use the Semgrep dashboard (beta) to gain an overview of your organization's security posture, including the deployment of guardrails.
+tags:
+  - Semgrep AppSec Platform
+  - Secure guardrails
+---
+
+# Dashboard (beta)
+
+The Semgrep dashboard (beta) is an overview of your organization’s security posture from data aggregated within Semgrep AppSec Platform. It helps you:
+
+- Evaluate your AppSec program, enabling you to know your current security risk.
+  - The dashboard beta includes assessing the deployment and adoption of **[secure guardrails](/secure-guardrails/secure-guardrails-in-semgrep)** to your organization.
```

Comment:
```suggestion
  - The dashboard (beta) assesses the deployment and adoption of **[secure guardrails](/secure-guardrails/secure-guardrails-in-semgrep)** to your organization.
```

Reply:
maybe "provides metrics to help you assess" or something like that because by itself the dashboard doesn't assess anything, the person does.